tests/test_md.py

```diff
@@ -884,3 +885,176 @@ def test_logging(tmp_path):
     assert log_file.exists()
     assert "emissions" in single_point.struct.info
     assert single_point.struct.info["emissions"] > 0
+
+
+def test_auto_restart(tmp_path):
+    """Test auto restarting simulation."""
+    # tmp_path for all files other than restart
+    # Include T300.0 to test Path.stem vs Path.name
+    final_path = tmp_path / "md-T300.0-final.extxyz"
+    traj_path = tmp_path / "md-T300.0-traj.extxyz"
+    stats_path = tmp_path / "md-T300.0-stats.dat"
+    log_file = tmp_path / "md.log"
+
+    # Predicted restart file, from defaults
+    restart_path = Path(".").absolute() / "NaCl-nvt-T300.0-res-4.extxyz"
```

Comment:
Do we want restart in the cwd or with output?

Reply:
Both, I think?

I use `tmp_path` for everything else, but since I don't specify a `file_prefix` or `restart_stem`, this tests the default restart file path, which will be created in the cwd, so that's where we want to restart from.

In other cases, e.g. if we set `file_prefix` with `tmp_pth`, the restart file should be created, and then used, in a different directory.

Reply:
Alright, just wanted to make sure. Sounds good.